cli/azd/pkg/alphafeatures/alpha_feature.go

```diff
@@ -0,0 +1,53 @@
+package alphafeatures
```

Comment:
I would prefer if we picked a shorter name. On my shortlist I have: `alpha` or `features` or `feat`

`alpha` is on my toplist, given that it's the identifying thing about the package, and I don't see how we would get into trouble here.

`alpha.features` is idiomatic
`alpha.<feature key>` seems fine as well.

Reply:
I think you should reconsider this feedback, @vhvb1989.  Right now, we end up with a lot of what go folks call "stuttering" in this package.  Consider the name of some of these types outside the package:

`alphafeatures.AlphaFeature`
`alphafeatures.IsAlphaKey`
`alphafeatures.AlphaFeatureManager`

Compare this to things from the go standard library like `http.Request` (instead of `http.HttpRequest`).

I think using `alpha` as a package name here would lead to names like the following:

`alpha.Feature`
`alpha.IsFeatureKey`
`alpha.FeatureManager`

This reads a lot better to me and ends up being shorter in code - a real help with our (very nice to have length limit to make it easy for GitHub reviews).

Reply:
that's 2 votes for `alpha` =D  . I will change it, but I am not convinced!!! hhhahaha

the example, `http` is a well known protocol name, it does not need more info.

the name `alpha` by itself, does not tell me much...  Imagine this is the first time you look a the code, and you see the folder `alpha` inside packages...  what does it mean?  

I guess I would prefer something like `featuretoggle` or `featurecontrol`.

Anyway, I will use this:

```
alpha.Feature
alpha.IsFeatureKey
alpha.FeatureManager
```

Reply:
updated it